MDX 파일에서는 eslint-plugin-prettier를 비활성화하고, MDX 내의 코드 블럭은 eslint-plugin-prettier로 포매팅하기 위한 패치
eslint-plugin-prettier는 기본적으로 MDX 린팅 과정에서 코드 블럭까지 전부 린팅하고,
이 때문에 코드 블럭 내의 코드가 입력으로 들어오면 그냥 무시해 버리도록 동작하지만,
이 레포에서는 MDX 린팅에 Prettier 대신 Remark를 사용 중이기 때문에 eslint-plugin-prettier는 MDX에 대해 비활성화됨
그러나 코드 블럭에 대해서는 Prettier를 사용하고 싶은데, 이를 위해서 코드 블럭을 무시하는 동작을 비활성화함

References:
  TODO: 업스트림에 이슈 작성 후 링크 추가 (아마 굉장히 niche한 요구사항이라 close되긴 할듯)

diff --git a/worker.mjs b/worker.mjs
index 1451209f44b626ef3e6f0b8c3e719db99fb791f8..0e2e6496eeeb96f9aca7a62b864b64bdb4e754a0 100644
--- a/worker.mjs
+++ b/worker.mjs
@@ -40,7 +40,7 @@
       : null;

     const { ignored, inferredParser } = await prettier.getFileInfo(
-      onDiskFilepath,
+      filepath,
       {
         resolveConfig: false,
         withNodeModules: false,
@@ -169,7 +169,7 @@
         'oxc-ts',
         'hermes',
       ];
-      if (parserBlocklist.includes(/** @type {string} */ (inferredParser))) {
+      if (false && parserBlocklist.includes(/** @type {string} */ (inferredParser))) {
         return;
       }
     }
